.gitattributes

```diff
@@ -1,2 +0,0 @@
-*.psd filter=lfs diff=lfs merge=lfs -text
-*.wav filter=lfs diff=lfs merge=lfs -text
```

Comment:
.gitattributes have already been removed from the latest `develop`. Could you delete the file from your branch just in case?

Reply:
![image](https://user-images.githubusercontent.com/42912708/74970871-1edfe180-545a-11ea-8251-7a7c55d1999b.png)

Been removed alr!